modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -19,18 +23,12 @@ GA:: General Availability
 
 | Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | |
 
+|1.16 | 0.62.x | 0.29.x | 0.38.x | 0.22.x (GA) | 1.18.x (TP) | 0.28.x (GA) | 0.11.x (TP) | 0.3.x (TP) | 4.15, 4.16, 4.17 | GA
```

Comment:
```suggestion
|1.16 | 0.62.x | 0.29.x | 0.38.x | 0.22.x (GA) | 1.18.x (TP) | 0.28.x (GA) | 0.12.x (TP) | 0.3.x (TP) | 4.15, 4.16, 4.17 | GA
```
I think results is 0.12.x @khrm  please confirm

Reply:
Yes, we ship Results 0.12.1